tests/SprykerEcoTest/Zed/Unzer/Business/UnzerFacade/ResolveMarketplacePaymentUnzerCredentialsTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Unzer\Business\UnzerFacade;
+
+use Generated\Shared\Transfer\UnzerMarketplacePaymentCredentialsResolverCriteriaTransfer;
+use SprykerEco\Shared\Unzer\UnzerConfig;
+use SprykerEcoTest\Zed\Unzer\Business\UnzerFacadeBaseTest;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Unzer
+ * @group Business
+ * @group UnzerFacade
+ * @group ResolveMarketplacePaymentUnzerCredentialsTest
+ */
+class ResolveMarketplacePaymentUnzerCredentialsTest extends UnzerFacadeBaseTest
```

Comment:
please check if we need to adjust tests for other affected facade methods I mentioned above

Reply:
Tests got already checked and adjusted if necessary.